apps/backoffice-v2/src/lib/blocks/variants/BlocksVariant/BlocksVariant.tsx

```diff
@@ -35,7 +37,7 @@ export const BlocksVariant: FunctionComponent<{
     return <ManualReviewBlocks />;
   }
 
-  if (isOngoingVariant) {
+  if (displayAmlBlockOnly) {
```

Comment:
Variants encompasses more then blocks or displays, I think the original name was more accurate.